src/http/client.rs

```diff
@@ -571,6 +733,52 @@ impl Http {
         }).await
     }
 
+    /// Edits a followup message for an Interaction.
+    ///
+    /// Refer to Discord's [docs] for Edit Webhook Message for field information.
+    ///
+    /// [docs]: https://discord.com/developers/docs/resources/webhook#edit-webhook-message
+    pub async fn edit_followup_message(
```

Comment:
```suggestion
    pub async fn edit_follow_up_message(
```

Reply:
discord calls them "followup messages" in their docs, so personally I would prefer to use that even if it is less grammatically correct :s